custom/icds_reports/tasks.py

```diff
@@ -1832,12 +1833,15 @@ def reconcile_data_not_in_ucr(reconciliation_status_pk):
             # These docs are invalid
             continue
         num_docs_retried += 1
-        not_found_in_es = doc_id in doc_ids_not_in_es
+        found_in_es = doc_id not in doc_ids_not_in_es
         if not inserted_at or (sql_modified_on - inserted_at).seconds > 3600:
             num_docs_unporcessed += 1
 
-        celery_task_logger.info(f'doc_id {doc_id} from {sql_modified_on} not found in UCR data sources. '
-            f'Not found in ES: {not_found_in_es}')
+        log = {
```

Comment:
Why json over single text line? I feel like the latter is often easier to grep, and json mixed into a general log file, probably still isnt super easy to automatically parse

Reply:
@calellowitz not quite sure what you mean, the JSON is a single text line. I just wanted to pull these into a text file and then do some processing in python so json seemed obvious. I agree it's not the best for processing in the shell without tools like `jq`.